### DISCUSSION TITLE
Durable Orchestration remains in running

### DISCUSSION THREAD

**Discussion by geovanegodoi** [Azure/azure-functions-durable-extension#2344] [Q&A]
I have developed an application using durable functions running within AKS, and tested it in SIT and HLG environments.

In these non production environments always an orchestration is finished its runtimeStatus turns to "completed".

However in the production environment, whenever the orchestration finishs its runtimeStatus always remains "running".

Environments SDK versions are the same.

I have done a search for similar discussions, but the solutions proposed did not work.

Any guess?

**Comment by davidmrdavid** [comment]
Hi @gigiogodoi. This is certainly a strange observation, though I'm unfortunately unable to suggest the root cause with the information provided.

@gigiogodoi: how are you able to determine that an orchestrator finishes in AKS if it's status remains in the "Running" state? Additionally, does this reproduce with a simple hello-world orchestrator?
